vault/logical_system_integ_test.go

```diff
@@ -64,7 +64,122 @@ func TestSystemBackend_enableAuth_plugin(t *testing.T) {
 	}
 }
 
-func TestBackend_PluginMain(t *testing.T) {
+func TestSystemBackend_PluginReload_plugin(t *testing.T) {
+	b, cleanup := testSystemBackendMock(t, 2)
+	defer cleanup()
+
+	req := logical.TestRequest(t, logical.UpdateOperation, "plugin/backend/reload")
+	req.Data["plugin"] = "mock-plugin"
+	resp, err := b.HandleRequest(req)
+	if err != nil {
+		t.Fatalf("err: %v", err)
+	}
+	if resp != nil {
+		t.Fatalf("bad: %v", resp)
+	}
```

Comment:
I realize this is making sure there's no error, but is there a way to verify that a reload actually happened other than no error?